app/views/actors/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<p id="notice"><%= notice %></p>
+
+<p>
+  <strong>Name:</strong>
+  <%= @actor.name %>
+</p>
```

Comment:
If would be nice to see the productions the actor has been in listed on this page.